features/error_reporting.feature

```diff
@@ -132,7 +132,6 @@ Feature: Error Reporting
           When an exception is thrown # features/exception_in_scenario.feature:7
             Exception: Exception is thrown in features/bootstrap/FeatureContext.php:56
             Stack trace:
-            #0 [internal function]: FeatureContext->anExceptionIsThrown()
```

Comment:
To fix the tests you need to apply a change like this to the scenario below

Reply:
That seems to have fixed it 🎉 

Weirdest thing though, locally it doesn't work 😕 

Locally I get
```
#1 src/Behat/Testwork/Call/Handler/RuntimeCallHandler.php(64): Behat\Testwork\Call\Handler\RuntimeCallHandler->executeCall()
```

instead of 
```
#1 src/Behat/Testwork/Call/Handler/RuntimeCallHandler.php(64): Behat\Testwork\Call\Handler\RuntimeCallHandler->executeCall(Object(Behat\Behat\Definition\Call\DefinitionCall))
```

(so `Object(Behat\Behat\Definition\Call\DefinitionCall)` is missing from the output)

but Travis and AppVoyer both have it in the output.

No idea why that is. I've tried with and without XDebug locally, doesn't matter.

I'm running PHP 7.4.3 in Ubuntu 20.04 in WSL2.

Reply:
@rpkamp: you probably have the php.ini setting `zend.exception_ignore_args` (introduced in PHP 7.4) to `On`, try turning it `Off` (source: https://stackoverflow.com/questions/60000391/php-missing-function-arguments-in-exception-stack-trace)

Reply:
I'll give that a whirl later, thanks! If that is indeed the case we should force that option to `Off` in the CI as well (if that isn't the case already).

Reply:
@guilliamxavier Yup, that was it 🙂 
I've created a PR to ensure this setting is off in CI: https://github.com/Behat/Behat/pull/1312